docs/utility-methods.md

```diff
@@ -194,30 +194,3 @@ q("*")
 
 // -> { cover: { ..., asset: { extension: string; mimeType: string; ...; metadata: { dimensions: { aspectRatio: number; height: number; width: number; }; }; }; } }[]
 ```
-
-## `makeContentBlockQuery`
-
```

Comment:
This is no longer needed, so I'm removing it here (because the change was causing TS errors, and I'm lazy and figured I'd just rip it out instead of fix it and then rip it out).